src/main/resources/config.properties

```diff
@@ -21,4 +22,11 @@ build.workspace.computer.not.found = Unable to access the computer for this buil
 matlab.command.build.step.name = runMATLABCommand
 matlab.tests.build.step.name = runMATLABTests
 matlab.command.step.display.name = Run MATLAB commands, scripts, or functions
-matlab.tests.step.display.name = Run MATLAB tests and generate artifacts
\ No newline at end of file
+matlab.tests.step.display.name = Run MATLAB tests and generate artifacts
+matlab.custom.location = Custom...
+matlab.added.to.path.from = Prepending MATLAB to PATH variable from this location: %s
+Axis.matlab.key = MATLAB
+Axis.use.matlab.warning = This project specifies MATLAB versions using the added 'MATLAB' axis as well as the 'Use MATLAB version' option. The values specified by 'Use MATLAB version' take precedence over the values specified by the 'MATLAB' axis.
+Axis.no.installed.matlab.error = Because the project does not have any MATLAB versions configured in Jenkins Global Tool Configuration, the plugin runs the default configuration.
+Use.matlab.version.axis.warning = This project specifies MATLAB version using the 'Use MATLAB version' option as well as the added 'MATLAB' axis. The values specified by 'Use MATLAB version' take precedence over the values specified by the 'MATLAB' axis.
```

Comment:
version or versions?
When can there be only one version? When can there be multiple versions?

Reply:
If a user has selected 'Use MATLAB Version' there will be only one version. If that's not the case and a MALTAB axis is added there can be versions of MATLAB.

I propose we change 'values' --> 'value' in second part of the sentence. Both the warning should then look like (Notice in case of warning that will be displayed at axis I use 'versions' in first sentence),

Use.matlab.version.axis.warning = This project specifies MATLAB version using the 'Use MATLAB version' option as well as the added 'MATLAB' axis. The value specified by 'Use MATLAB version' take precedence over the values specified by the 'MATLAB' axis.

Axis.use.matlab.warning = This project specifies MATLAB versions using the added 'MATLAB' axis as well as the 'Use MATLAB version' option. The value specified by 'Use MATLAB version' take precedence over the values specified by the 'MATLAB' axis.